Gems/Multiplayer/Code/Source/Components/LocalPredictionPlayerInputComponent.cpp

```diff
@@ -223,6 +223,7 @@ namespace Multiplayer
                 }
                 else
                 {
+#ifndef AZ_RELEASE_BUILD
```

Comment:
If we are doing ```AZ_RELEASE_BUILD=1``` in the cmake file, then this should change to an #if instead. ```AZ_RELEASE_BUILD=1``` could be misleading and someone reading it may assume ```AZ_RELEASE_BUILD=0``` means to turn it off, which #ifdef will not handle it as expected
```suggestion
#ifndef AZ_RELEASE_BUILD
```
```suggestion
#if !AZ_RELEASE_BUILD
```